dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -1,7 +1,7 @@
 HikariCP/2.5.1//HikariCP-2.5.1.jar
 JLargeArrays/1.5//JLargeArrays-1.5.jar
 JTransforms/3.1//JTransforms-3.1.jar
-RoaringBitmap/0.9.45//RoaringBitmap-0.9.45.jar
+RoaringBitmap/0.9.46//RoaringBitmap-0.9.46.jar
```

Comment:
I think we should probably have a buferred update within individual release thread. Upgrading is good and a legitimate improvement but could be done in batch within each release window.

Reply:
+1 to @HyukjinKwon's

Reply:
Maybe once a month? Or two months?